scriptmodules/helpers.sh

```diff
@@ -131,6 +132,57 @@ function editFile() {
     [[ -n "$choice" ]] && echo "$choice" >"$file"
 }
 
+## @fn configKernelModule()
+## @brief mode configuration mode (install or remove)
+## @brief module module name to be added to /etc/modules-load.d file
+## @param prefix prefix name of .conf files in /etc/modprobe.d & /etc/modules-load.d
+## @param parameter module parameter to be added to /etc/modprobe.d file
+function configKernelModule() {
+    local mode="$1"
+    local module="$2"
+    local prefix="$3"
+    local parameter="$4"
+
+    [[ -z "$prefix" ]] && prefix="$module"
+
+    local parameter_file="/etc/modprobe.d/$prefix.conf"
+    local module_file="/etc/modules-load.d/$prefix.conf"
+
+    # SysVinit legacy compatibility
+    [[ ! -d "/etc/modules-load.d/" ]] && module_file="/etc/modules"
+
+    if [[ "$mode" == "remove" ]]; then
+        # remove all entries matching module name from configs
+        touch "$module_file" "$parameter_file"
+        sed -i --follow-symlinks "/$module\b/d" "$module_file" "$parameter_file"
+        # delete empty config files (if necessary)
+        [[ ! -s "$module_file" && -d "/etc/modules-load.d/" ]] && rm -f "$module_file"
```

Comment:
Not sure about this logic - I don't think it's too clear (and checking if the dir exists before removing module file etc)